Makefile

```diff
@@ -56,8 +58,13 @@ prepare-tests-files:
 	python -m spacy link de_core_news_sm de --force
 	wget --progress=dot:giga -N -P data/ https://s3-eu-west-1.amazonaws.com/mitie/total_word_feature_extractor.dat
 
-test: clean
-	py.test tests --cov rasa
+test: clean get-num-jobs
+	# OMP_NUM_THREADS can improve overral performance using one thread by process (on tensorflow), avoiding overload
+	OMP_NUM_THREADS=1 pytest tests -n $(JOBS) --cov rasa
+
+get-num-jobs:
+	$(eval JOBS := $(if $(findstring -j, $(MAKEFLAGS)), $(shell echo $(MAKEFLAGS) | sed -E "s@.*-j([0-9]+).*@\1@"), $(JOBS)))
+	$(eval JOBS := $(if $(findstring -j, $(JOBS)), auto, $(JOBS)))
```

Comment:
what are these two needed for?

Reply:
It's because when user pass the flag `-j` to Make, specifying the desired number of jobs, this information is not passed to pytest and hence tests would still be run sequentially. Without these lines, the number of jobs that pytest would use would always be the same (or auto) when called with `make test`, and for example `make test -j 2` wouldn't have any effect (on pytest).